NEWS.md

```diff
@@ -2,6 +2,58 @@
 
 ## Unreleased
 
+## v0.33.0
+
+### What's New
+
+- **Arweave file data sources** - Arweace file data sources allow subgraph developers to query offchain information from Arweave directly in mappings.[(#4789)](https://github.com/graphprotocol/graph-node/pull/4789)
```

Comment:
```suggestion
- **Arweave file data sources** - Arweave file data sources allow subgraph developers to query offchain information from Arweave from within the subgraph mappings.[(#4789)](https://github.com/graphprotocol/graph-node/pull/4789)
```

Reply:
I think "access" rather than "query" might be clearer here